OOModel/src/declarations/Class.cpp

```diff
@@ -228,4 +239,22 @@ Class* Class::expressionToClass(Expression* expr)
 	return nullptr;
 }
 
+QSet<Class*> Class::directSubClasses()
+{
+	auto top = root();
+	//Find all the classes where this method is subclasses
+	QSet<Class*> result;
+	QList<Model::Node*> toCheck;
+	toCheck.append(top);
+	while (!toCheck.isEmpty())
```

Comment:
This loop is rather expensive. I'm wondering if there is a way to make it quicker. Any thoughts?